server/hap2/test/TestHaplib.py

```diff
@@ -174,3 +174,39 @@ def test_setup(self):
                             "amqp_password": "test_password"}
         rabbitmq_connector = haplib.RabbitMQHapiConnector()
         common.assertNotRaises(rabbitmq_connector.setup, transporter_args)
+
```

Comment:
2 blank lines are recommended:

https://www.python.org/dev/peps/pep-0008/#blank-lines

In addition, pep8 command reports some issues.
Please check them.